FE/index.html

```diff
@@ -17,7 +17,7 @@
       <div id="searchDiv">
```

Comment:
esse nome podia ter Div no id é redundante
```suggestion
      <div id="searchTasks">
```